src/screens/Signin.tsx

```diff
@@ -16,6 +18,30 @@ const SigninScreen = ({ navigation }: any) => {
         style={styles.button}
         onPress={() => navigation.navigate("Home")}
       />
+      <TextInput 
+        placeholder="email"
+        onChangeText={(email) => setEmail(email)}
+      />
+      <TextInput
+        placeholder="password"
+        onChangeText={(pass) => setPass(pass)}
+      />
+      <Button
+        title="Sign Up"
+        style={styles.button}
+        onPress={async () => {
```

Comment:
Can you extract this code into a function above the return statement? That will make this code a little cleaner and easier to add on to if we need to later on. This line would look like onPress = {() => functionCall()}

Reply:
Yes! Just committed the change. Also, included the getAllUsers button as well, to provide testing for that.